reference_data/tests_only/spherical_expansion_gradient_test.json

```diff
@@ -2,9 +2,9 @@
     "direction_mode": "Cartesian",
     "comment": "Symmetry directions below for diamond to separate radial and angular components (set direction_mode to Provided to use)",
     "displacement_directions": [
-      [1.0, 1.0, 1.0],
-      [1.0, 0.0, -1.0],
-      [-1.0, 2.0, -1.0]
+      [1.0, 0, 0],
+      [0, 1, 0],
+      [0, 0, 1]
```

Comment:
This is unnecessary; the `"direction_mode": "Cartesian"` above already takes care of this